src/background/background.coffee

```diff
@@ -74,7 +74,7 @@ handleRequestFromContentScript = (request, sender, sendResponse) ->
     if script.indexOf('Shim') == -1
       trackController(request.host)
     chrome.tabs.executeScript(sender.tab.id, {file: script})
-    if request.host.indexOf("youtube") > -1
+    if request.host.indexOf("youtube") > -1 || request.host.indexOf("connect.monstercat") > -1
```

Comment:
This is a bad hack and should be dealt with before more services rely on this Shim + Controller combo